### ISSUE TITLE
Figure 3.71.4.1.2.2-1: Contains PlantUML code

### ISSUE THREAD

**Issue by DavidPyke** [IHE/ITI.IUA#82]
**Section Number** *Identify the most specific section number the issue occurs (e.g. 4.1.2)*
Figure 3.71.4.1.2.2-1: Sequence of HTTP requests in the client authorization code grant type

**Issue** *Describe your issue. Don't write a book, but do include enough to indicate what you see as a problem.*
The plantUML code for the figure is shown below the figure.

**Proposed Change** *Propose a resolution to your issue (e.g., suggested new wording or description of a way to address the issue). The committee might simply accept your suggested text. Even if they don't, it gives a good sense of what you are looking for. Leaving this blank means you can't imagine how to resolve the issue, which makes it easier for the committee to admit they can't imagine how to resolve it either and leave it unresolved.*
Remove the code

**Priority**:
- Low: Typo or other minor classification that an editor can manage. Requires no group discussion.

**Comment by msmock** [comment]
Yes. That was intended. Did to policy change to not show the plantuml ?

**Comment by DavidPyke** [comment]
If someone needs the plantuml code, they can get it from the repository.  It shouldn't be in the published document

**Comment by JohnMoehrke** [comment]
If we revise IUA for other reasons, we should remove the inline plantuml and replace it with a link to the plantuml file in github.
